### PR TITLE
Media isolation.

### PR DESCRIPTION
**Pull request by guillaume-brossard-adsk**
Not to merge until all the sections have been written.

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/59a3945c7c35c82a8ec51493851b33c8d87fa961/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/55b69aad1a7b64c0868643ad7086529f255e054e/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/2e8a5884b11b5d7e45f337d7b9540149d5c4514c/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/39657ee905d6d24c4451f5b2698764dde8dff085/index.html)

File: docs/en/trusted-solutions/tier1/features/media_isolation.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+<img alt="media-isolation-overview" src="../images/media-isolation-overview.png" width="400">
+
+## Client Owned S3 Bucket
+Storing media and attachments in a S3 bucket you own allow you to remain the legal owner of these artifacts, allowing you to comply with your security and legal intermal policies. Your studio remains in control of the assets and the access to the assets, access that you can revoke at will.
+
+<img alt="media-isolation-arch" src="../images/media-isolation-arch.png" width="400">
+
+## More about Access
+When using Shotgun, media is downloaded and uploaded from/to S3, without transitting through Shotgun infrastructure. Shotgun will only get access to the media in two circonstances.
```
Comment (rlyne) [inline docs/en/trusted-solutions/tier1/features/media_isolation.md:19]: When using Shotgun to upload and download media it is transferred directly to / from AWS S3 without transiting through Autodesk infrastructure. Shotgun will only access media in two circumstances:

File: docs/en/trusted-solutions/tier1/features/media_isolation.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+1. The Shotgun Transcoding service will get read/write access once, soon after upload, when transcoding the media. See [Ephemeral Transcoding](../getting_started/about.md#ephemeral-transcoding) for details.
+2. When the Shotgun service generates S3 Links to your sources and transcoded media.
+
+This is rendered possible by leveraging [AWS AssumeRole keyless Security Token Service](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html). When setting up Media Isolation, an AWS Role allowing Shotgun to access your media for the action listed above is created, and the Shotgun service is allowed to assume that role.
+
+Shotgun Support staff doesn't have access to the S3 Bucket under any circonstances.
```
Comment (rlyne) [inline docs/en/trusted-solutions/tier1/features/media_isolation.md:25]: Shotgun Support staff do not have access to your S3 Bucket under any circumstances.

File: docs/en/trusted-solutions/tier1/features/media_isolation.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Costs
+When activating Media Isolation, the following costs, usually covered by Autodesk, becomes the responsibility of the client.
```
Comment (rlyne) [inline docs/en/trusted-solutions/tier1/features/media_isolation.md:28]: When activating Media Isolation the following costs, previously covered by Autodesk, become the responsibility of the client:

File: docs/en/trusted-solutions/tier1/features/media_isolation.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+1. **S3 Costs.** All the S3 storage costs will be assumed by the customer. See [Media Isolation](../setup/tuning.md) for more details about how to reduce costs.
+2. **S3 Bandwidth.** Bandwidth out of the S3 bucket will be assumed by the customer.
+
+## What Media Isolation is not providing
+Activating media isolation doesn't guarantee that the access to your site or media is done in closed network.
```
Comment (rlyne) [inline docs/en/trusted-solutions/tier1/features/media_isolation.md:33]: Activating Media Isolation doesn't guarantee that the access to your Shotgun site or media takes place within a closed network.

File: docs/en/trusted-solutions/tier1/features/media_replication.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+Shotgun is compatible with the S3 Cross-Region replication feature, allowing your users located in different regions to read from the region closer to them in order to reduce latency and increase throughput. Replication to one region is currently supported.
+
+<img alt="media-replication-overview" src="../images/media-replication-overview.png" width="400">
+
+## Pre-requisites
+Media Isolation is required in order to elect Media Replication.
+
+## Configuration by users
+When using media replication, each users can customize from which region data is read from. A user can either specify the region to use, or use the automatic mode. In automatic mode, a user will use the replica if it is amongst the IP range specified in the Isolation Preferences.
```
Comment (rlyne) [inline docs/en/trusted-solutions/tier1/features/media_replication.md:18]: When using Media Replication, each user can customize which region data is read from. A user can either specify the region to use, or use automatic mode. In automatic mode Shotgun selects the replica determined by the user's IP address using IP ranges specified in the Isolation Preferences.

**Review by rlyne — COMMENTED** [review]
All the feature pages apart from Media Isolation have a Next Steps section at the bottom linking to the relevant setup page. It would be good to have this for consistency.

File: docs/en/trusted-solutions/tier1/features/media_traffic_isolation.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+<img alt="media-traffic-isolation-overview" src="../images/media-traffic-isolation-overview.png" width="400">
+
+## Configuration
+An S3 Proxy component is deployed within your VPC which is then used as the endpoint for all S3 communication. It can also be made publicly available using AWS Global Accelerator. 
+
+## How it works
+Shotgun can be configured to use an S3 Proxy address to communicate with your S3 bucket. Deploying the S3 Proxy component within your VPC makes it possible to isolate traffic from the public Internet completely, or to allow more tightly controlled access from the Internet to your media.
+
+<img alt="media-traffic-isolation-arch" src="../images/media-traffic-isolation-arch.png" width="400">
```
[the remark this answers is not in the document]
Comment (guillaume-brossard-adsk) [reply docs/en/trusted-solutions/tier1/features/media_traffic_isolation.md:20]: I'll increase it a bit, but if the user right-click + View image he can get the image in higher definition...

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/a5e5c3668efa1e449958279cf73f6e3290809dd6/index.html)

File: docs/en/trusted-solutions/tier1/features/web_traffic_isolation.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+## Configuration
+A unique VPC endpoint is provided by Autodesk. Access to the endpoint must be configured from your VPC and the relevant DNS configuration completed to allow your client systems to reach the new endpoint. 
+
+## How it works
+By using the provided VPC endpoint web traffic is isolated from the public Internet completely. All web traffic transits between your AWS VPC and Autodesk's AWS VPC rather than the public Internet.
+
+<img alt="web-traffic-isolation-arch" src="../images/web-traffic-isolation-arch.png" width="400">
+
+## Costs
+The only cost associated with Web Traffic Isolation are those related to web traffic transiting your AWS VPC. See [AWS VPC Pricing](https://aws.amazon.com/vpc/pricing) for more details.
```
Comment (fchristopher) [inline docs/en/trusted-solutions/tier1/features/web_traffic_isolation.md:23]: should this be changed to `transiting to/from your AWS VPC`?

**Review by fchristopher — APPROVED** [review]
Suggested a few grammatical changes - punctuation, reworking of sentences, etc.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/63eb9068d3a461be89003da4cbcda36e8869f494/index.html)

**Pull request merged**